java-client/src/test/java/org/opensearch/client/opensearch/integTest/restclient/CatIT.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.client.opensearch.integTest.restclient;
+
+import org.apache.hc.core5.http.HttpHost;
```

Comment:
`import org.apache.http.HttpHost;`

Reply:
@MarinaRazumovsky I will take care of it, thank you

Reply:
Thanks both! Next time feel free to open another PR, might be faster than waiting for a maintainer to amend/fix/force push.